controllers/fenceagentsremediation_controller_test.go

```diff
@@ -69,39 +72,53 @@ var _ = Describe("FAR Controller", func() {
 	underTestFAR = newFenceAgentsRemediation(validNodeName, fenceAgentIPMI, testShareParam, testNodeParam)
 
 	Context("Functionality", func() {
-		Context("buildFenceAgentParams", func() {
-			When("FAR's name isn't a node name", func() {
+		Context("buildFenceAgentParams - check CR name", func() {
```

Comment:
Nit: using function names in Ginkgo's node description makes it harder to understand what the test is verifying.
For example, in this context, it might be just "Check CR name against node"

Reply:
Ok, do you have a suggestion for better naming?
I see a small value of testing the 2 functions in UT, but it does catch errors when I do small changes in them (or their dependencies)

Reply:
For example, in this context, it might be just "Check CR name against node"

Reply:
I see, then I tend to let the context just be the function name (`buildFenceAgentParams`), and just remove the ` - check CR name`

Reply:
Indeed, checking behavior rather than a function (by it's name) seems more right.
Something to think off for next PRs and writing UTs